src/Sarif/FileRegionsCache.cs

```diff
@@ -33,10 +34,10 @@ public FileRegionsCache(Run run, int capacity = DefaultCacheCapacity, IFileSyste
 
         /// <summary>
```

Comment:
>/// [](start = 8, length = 3)

Suggestion: Place the XML doc comments on the interface methods, and mark the implementations with `/// <inheritdoc/>`.

Reply:
just updated!

---
In reply to: [511616376](https://github.com/microsoft/sarif-sdk/pull/2130#discussion_r511616376) [](ancestors = 511616376)

Reply:
Let's provide XML doc comments for both `public` methods, while we're in here. Don't forget to fill in descriptions of each parameter, and also the return value.

---
In reply to: [511620145](https://github.com/microsoft/sarif-sdk/pull/2130#discussion_r511620145) [](ancestors = 511620145,511616376)

Reply:
Just updated! Can you check if that is good enough?

---
In reply to: [511622336](https://github.com/microsoft/sarif-sdk/pull/2130#discussion_r511622336) [](ancestors = 511622336,511620145,511616376)